addons/mail/static/src/models/rtc_session/rtc_session.js

```diff
@@ -147,28 +146,15 @@ registerModel({
                 return;
             }
             this.update(data);
-            this._debounce(async () => {
-                if (!this.exists()) {
-                    return;
-                }
-                await this.async(() => {
-                    this.env.services.rpc(
-                        {
-                            route: '/mail/rtc/session/update_and_broadcast',
-                            params: {
-                                session_id: this.id,
-                                values: {
-                                    is_camera_on: this.isCameraOn,
-                                    is_deaf: this.isDeaf,
-                                    is_muted: this.isSelfMuted,
-                                    is_screen_sharing_on: this.isScreenSharingOn,
-                                },
-                            },
-                        },
-                        { shadow: true }
-                    );
-                });
-            }, 3000);
+            if (this.broadcastTimeout) {
+                browser.clearTimeout(this.broadcastTimeout);
```

Comment:
Unrelated to diff on this PR, but shouldn't this flow be changed to a throttle rather than a debounce?
If values keeps changing, the update will never happen as the previous timeout is always cleared.